app/studio/1.0.x/index.md

```diff
@@ -2,19 +2,64 @@
 title: Kong Studio
 ---
 
-![Kong Studio](https://doc-assets.konghq.com/studio/1.0/kong-studio-welcome.png)
+Kong Studio's base Insomnia Designer enables spec-first development for all REST and GraphQL services. 
```

Comment:
```suggestion
Kong Studio is an Enterprise module based on Insomnia Designer, which enables spec-first development for all REST and GraphQL services. 
```
Not sure if "module" is the right term here, but I'd suggest calling it out in some way.

Reply:
Let's use `bundle`